net/interfaces/interfaces.go

```diff
@@ -539,7 +573,8 @@ func anyInterestingIP(pfxs []netaddr.IPPrefix) bool {
 // should log in interfaces.State logging. We don't need to show
 // localhost or link-local addresses.
 func isInterestingIP(ip netaddr.IP) bool {
-	if ip.IsLoopback() || ip.IsLinkLocalUnicast() {
+	if ip.IsLoopback() ||
+		(ip.Is4() && ip.IsLinkLocalUnicast() && hostinfo.GetEnvType() == hostinfo.AWSLambda) {
```

Comment:
This part of the change seems unnecessary for 1.10. It only controls the interface formatting, and only makes it print less. Removing this doesn't prevent any lambda things from working, right?

Reply:
Removed for 1.10.